File: client/src/app/_models/comicNewestDto.ts
import { ChapterForComicNewestDto } from './chapterForComicNewestDto';

export interface ComicNewestDto {
  id: number;
  comicName: string;
  isFeatured: boolean;
  mainImage: string;
  noFollows: number;
  noViews: number;
  rate: number;
  noReviews: number;
  noComments: number;
  chapters: ChapterForComicNewestDto[];
}

File: API/Dtos/ComicNewestDto.cs
namespace API.Dtos
{
    public class ComicNewestDto
    {
        public int Id { get; set; }
        public string ComicName { get; set; }
        public bool IsFeatured { get; set; }
        public string MainImage { get; set; }
        public int Rate { get; set; }
        public int NOReviews { get; set; }
        public int NOViews { get; set; }
        public int NOFollows { get; set; }
        public int NOComments { get; set; }
        public List<ChapterForComicNewestDto> Chapters { get; set; }
    }
}